functions/lib/pubsub/processUploads.js

```diff
@@ -152,3 +158,40 @@ function compressImage(tempFilePath, imageManipulation) {
     return imageManipulation.compress(tempFilePath, 960);
   }
 }
+
+function readImageData(mediaPath, localFS) {
+  return function() {
+    const imageData = {}
+    return localFS.readFile(mediaPath)
+      .then((data) => {imageData.mediaData = data})
```

Comment:
Oh awesome, did not know this. That works and makes it much simpler. Fixed and removed the code built around the call `fs.stat` call.